dags/data_contributions.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+import airflow.operators
+from airflow.models import DAG
+import utils.helpers as helpers
+
+args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'start_date': datetime.datetime(2017, 3, 1),
+    'retries': 1,
+    'retry_delay': datetime.timedelta(minutes=10),
+}
+
+dag = DAG(
+    dag_id='data_contributions',
+    default_args=args,
+    max_active_runs=1,
+    schedule_interval='@daily'
+)
+
+unprocessed_data_contributions_sensor_task = airflow.operators.SqlSensor(
+    task_id='unprocessed_data_contributions_sensor',
+    dag=dag,
+    conn_id='explorer_db',
+    sql='SELECT id FROM data_contributions WHERE approved=true AND document_id IS NULL'
```

Comment:
@vitorbaptista  As I'm sure you noticed [from these 2 queries](https://github.com/opentrials/processors/blob/707b0f5207343e37bda261b5882058e0f3ba2c6e/processors/data_contributions/processor.py#L29-L44), we are interested to process `data_contributions` even if they already have `document_id` because they could be updated. For example they could be assigned to another `trial`.
So, the ideal trigger would start `data_contributions` processor every time a `data_contribution` is updated.
From what I understand, with this trigger,  updated `data_contributions` that already have a `document_id`  would have to wait until: a new data contribution is added + a curator finds the time to approve the new contribution. This could be quite long. 
So,  why don't we just run the `data_contributions` processor daily? It wouldn't be too consuming since there's very few `data_contributions` and even those are filtered in the processor.